### ISSUE TITLE
[Design] 10-10EZ - SIGI: Confirm current content and make any necessary updates

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#46904]
**BLOCKED**: Due to pending decision on content with stakeholders

## Background

The Self-Identifying Gender Identity feature has been on hold for about a year.  It has been finally scheduled for a November release for the Enrollment System.  
With this news, and the past year of deep research on gender identities and interviews done by multiple teams within the VA program, we need to confirm that the design and content we have aligns with the direction of VA.gov ideals and patterns.

## Tasks
- [x] Confirm with Content team on current design and content of the SIGI page
- [x] Make any necessary changes
     - [x] Update the team on any changes to review
     - [x] Create FE ticket for changes needed to code

## Acceptance Criteria
- Design and content aligns with VA.gov direction for Self-Identifying Gender Identity patterns

**Comment by jessicastump** [comment]
[Content Review Meeting 9/9/22 Recap](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25350#issuecomment-1243983617)

Sketch Designs

- [Birth Sex - Desktop (v001)](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/3ORElvo)
- [Birth Sex - Mobile (v001)](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/YGV83lO)
- [Birth Sex - Desktop (v002)](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/DPjE4qj)
- [Birth Sex - Mobile (v002)](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/QbAM548)

- [Gender Identity - Desktop ](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/ag9MnWd)
- [Gender Identity - Mobile](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/Kvem2WK)

Waiting confirmation from content team on meeting recap, then ready for final review.

**Comment by hdjustice** [comment]
Per Slack conversation (below), it was decided to create a 2-part question, starting with something like "Do you want to share your gender identity?" and if so, an additional question with the options.  
For FE work,  we will need to confirm with Joshua & team on the data being sent to ES.  I believe we will only want to pass the option selected, because the ES downstream will not be able to accept the Yes/No answer to the first question.  However, It may be possible to translate the No answer to "prefer not to disclose".

Slack conversation
>**Heather Justice**
>Hi Patrick - Is there anything more on SIGI that you want to review or update before we start with updating FE content and QA?
>
>**Patrick Bateman**
>hi! yes, there is, and kind of out of left field, so thank you for the prompt.
>tl;dr, can we make it a two part question?
>first question is "do you want to disclose your self identified gender identity?"
>if yes, then next question contains the choices (minus "prefer not to disclose" maybe)
>if no, we code their response for ES purposes as "prefer not to disclose", and that is it.
>cc Danielle Thierry to make sure im remember that right from our convo yesterday
>
>**Danielle Thierry**
>Yep! I would just change "disclose" to "share." And then we want to keep the "what to know" attached to the first >question.
>
>**Heather Justice**
>Alrighty, thanks for this!  
>Jessica Stump - Any concerns here?
>
>**Patrick Bateman**
>to be clear, language i used above is just short hand, not exactly what the words should be
>
>**Jessica Stump**
>No concerns here, I will get the design updated to reflect that and send out updated links.

**Comment by jessicastump** [comment]
Here are the Sketch links to the updated designs:

- [Gender identity page 1 (yes/no share question) - Desktop](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/qemoRK2)
- [Gender identity page 1 - Mobile](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/Jn4075k)
- [Gender identity page 2 (select) - Desktop](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/wLkE92q)
- [Gender identity page 2 - Mobile](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/ZOYPb93)

**Comment by jessicastump** [comment]
[Front end ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47939)

**Comment by jessicastump** [comment]
Sketch links aligning with design system gender pattern - single/optional question, [shared 10/12](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/product/Bi-weekly%20Stakeholder%20Mtg%20notes/10-12-22%20HEC%20Bi-weekly%20Health%20Care%20app%20update.md#101222-agenda-items):
- [Desktop](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/l1aLd5K)
- [Mobile](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/VrLb0m7)

**Comment by hdjustice** [comment]
Per our [bi-weekly meeting with the HEC team](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/product/Bi-weekly%20Stakeholder%20Mtg%20notes/10-12-22%20HEC%20Bi-weekly%20Health%20Care%20app%20update.md) (Tarsha and Christopher Hawkins), a concern was expressed that the Gender options did not match the paper form (Man vs Male, Woman vs Female).  We stated our case, provided a screenshot and talking points to Tarsha for discussing with the HEC leadership.

**Comment by hdjustice** [comment]
This issue is blocked due to pending decisions on Content between VA.gov and Stakeholders.  We will proceed with Accessibility Testing and Staging review once the decision is made and updates are complete.

**Comment by hdjustice** [comment]
Closing all SIGI tickets - There has been no further movement or discussion on this topic
